src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -87,6 +99,39 @@ export const OAuthManagement: React.FunctionComponent = () => {
           isShowingConfirmModal={isShowingConfirmModal}
         />
       )}
+      <Modal
+        show={isShowingVerification}
+        animation={false}
+        backdrop="static"
+        onHide={() => setIsShowingVerification(false)}
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>
+            <Typography variant="headline1">Submit Verification</Typography>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <p>
+            In order to verify an OAuth client please send an email to{' '}
```

Comment:
Now that we use the service desk, please update these instructions to
In order to verify an OAuth client please submit a request to the Synapse Service Desk (with a link to
https://sagebionetworks.jira.com/servicedesk/customer/portal/9)